File: Dream.WebApp/Models/SecuritizationModel.cs
using System;

namespace Dream.WebApp.Models
{
    public class SecuritizationModel
    {
        public string SecuritizationDescription { get; set; }
        public int SecuritizationDataSetId { get; set; }
        public int SecuritizationVersionId { get; set; }
        public SecuritizationSummaryModel SecuritizationSummaryModel { get; set; }
        public SecuritizationNodeModel SecuritizationNodeModel { get; set; }
        public SecuritizationTrancheModel SecuritizationTrancheModel { get; set; }
        public FeeTrancheModel FeeTrancheModel { get; set; }
        public ReserveAccountModel ReserveAccountModel { get; set; }
        public PriorityOfPaymentsModel PriorityOfPaymentsModel { get; set; }
        public MarketRateEnvironmentModel MarketRateEnvironmentModel { get; set; }
        public PerformanceAssumptionsModel PerformanceAssumptionsModel { get; set; }
        public ScenarioOptionsModel ScenarioOptionsModel { get; set; }
        public PaceAssessmentRecordModel PaceAssessmentRecordModel { get; set; }
        public CollateralizedTranchesModel CollateralizedTranchesModel { get; set; }
        public DateTime CollateralCutOffDate { get; set; }
        public DateTime CashFlowStartDate { get; set; }
        public DateTime InterestAccrualStartDate { get; set; }
        public DateTime SecuritizationClosingDate { get; set; }
        public DateTime SecuritizationFirstCashFlowDate { get; set; }
        public string RedemptionLogicDescription { get; set; }
        public string NominalSpreadRateIndexGroup { get; set; }
        public string CurveSpreadRateIndexGroup { get; set; }
        public bool IsResecuritization { get; set; }
        public bool UseReplines { get; set; }
        public bool SeparatePrepaymentInterest { get; set; }
        public double? PreFundingAmount { get; set; }
        public double? PreFundingBondCount { get; set; }
        public double? CleanUpCallPercentage { get; set; }
    }
}

File: Dream.WebApp/app/classes/models/securitizationModel.ts
import { SecuritizationSummaryModel } from './securitizationSummaryModel';
import { SecuritizationNodeModel } from './securitizationNodeModel';
import { SecuritizationTrancheModel } from './securitizationTrancheModel';
import { FeeTrancheModel } from './feeTrancheModel';
import { ReserveAccountModel } from './reserveAccountModel';
import { PriorityOfPaymentsModel } from './priorityOfPaymentsModel';
import { MarketRateEnvironmentModel } from './MarketRateEnvironmentModel';
import { PerformanceAssumptionsModel } from './performanceAssumptionsModel';
import { ScenarioOptionsModel } from './scenarioOptionsModel';
import { PaceAssessmentRecordModel } from './paceAssessmentRecordModel';
import { CollateralizedTranchesModel } from './collateralizedTranchesModel';

export class SecuritizationModel {
    SecuritizationDescription: string;
    SecuritizationDataSetId: number;
    SecuritizationVersionId: number;
    SecuritizationSummaryModel: SecuritizationSummaryModel;
    SecuritizationNodeModel: SecuritizationNodeModel;
    SecuritizationTrancheModel: SecuritizationTrancheModel;
    FeeTrancheModel: FeeTrancheModel;
    ReserveAccountModel: ReserveAccountModel;
    PriorityOfPaymentsModel: PriorityOfPaymentsModel;
    MarketRateEnvironmentModel: MarketRateEnvironmentModel;
    PerformanceAssumptionsModel: PerformanceAssumptionsModel;
    ScenarioOptionsModel: ScenarioOptionsModel;
    PaceAssessmentRecordModel: PaceAssessmentRecordModel;
    CollateralizedTranchesModel: CollateralizedTranchesModel;
    CollateralCutOffDate: Date;
    CashFlowStartDate: Date;
    InterestAccrualStartDate: Date;
    SecuritizationClosingDate: Date;
    SecuritizationFirstCashFlowDate: Date;
    RedemptionLogicDescription: string;
    NominalSpreadRateIndexGroup: string;
    CurveSpreadRateIndexGroup: string;
    IsResecuritization: boolean;
    UseReplines: boolean;
    SeparatePrepaymentInterest: boolean
    PreFundingAmount: number;
    PreFundingBondCount: number;
    CleanUpCallPercentage: number;
}